tests/testing_utils.py

```diff
@@ -232,3 +234,36 @@ def full_path(self, filename):
         f.full_path('foo/bar.txt') -> /tmp/asdfasd/foo/bar.txt
         """
         return os.path.join(self.rootdir, filename)
+
+
+def _get_current_account_id():
+    sts = boto3.client("sts")
+    account_id = sts.get_caller_identity()["Account"]
+    return account_id
+
+
+class UpdatableSARTemplate:
+    def __init__(self, source_template_path):
+        self.source_template_path = source_template_path
+        self.temp_directory = tempfile.TemporaryDirectory()
+        self.temp_directory_path = Path(tempfile.TemporaryDirectory().name)
+        self.updated_template_path = None
+
+    def setup(self):
+        with open(self.source_template_path, "r") as sar_template:
+            updated_template_content = sar_template.read()
+        updated_template_content = updated_template_content.replace("${AWS::AccountId}", _get_current_account_id())
```

Comment:
For my understanding (I can't see the test failure logs for some reason) why do we need to do this manually instead of letting the intrinsic sub handle it?

Reply:
The issue we do not support Sub intrinsic function :( .. This is the error we got in the canaries:
```
Error: [InvalidResourceException('MyApplication', "Property 'ApplicationId' cannot be resolved. Only FindInMap and Ref intrinsic functions are supported.")] ('MyApplication', "Property 'ApplicationId' cannot be resolved. Only FindInMap and Ref intrinsic functions are supported.")
```

Reply:
Can we add a comment about this so we won't revert it by mistake in the future?

Reply:
done